project/Dependencies.scala

```diff
@@ -24,6 +24,8 @@ object Dependencies {
     .withRank(KeyRanks.Invisible) // avoid 'unused key' warning
 
   val junitVersion = "4.13.2"
+  val junit5Version = "5.9.3"
```

Comment:
can you use junit 5.10.0?

A rebase of the PR would be useful too. We are now in a position to be able to merge v1.1.0 target PRs.